app/controllers/idv/ssn_controller.rb

```diff
@@ -1,11 +1,11 @@
 module Idv
   class SsnController < ApplicationController
     include IdvSession
+    include IdvStepConcern
     include StepIndicatorConcern
     include StepUtilitiesConcern
     include Steps::ThreatMetrixStepHelper
 
-    before_action :confirm_two_factor_authenticated
```

Comment:
Given that this method is defined in IdvSession, but auto-included as a before action in IdvStepConcern (which in turn includes IdvSession), should IdvSession be responsible for auto-including this before action?